app/models/reservation.rb

```diff
@@ -0,0 +1,3 @@
+class Reservation < ApplicationRecord
```

Comment:
well done @khanh-devos  but I thing this model is supposed to be a many to many relation so we need to link it to a user a reservation belongs to a user and a belongs to a bike, I also think  that the reservation date is missing here